hummingbird/ml/containers/sklearn/pytorch_containers.py

```diff
@@ -80,6 +80,7 @@ def save(self, location):
             with open(os.path.join(location, constants.SAVE_LOAD_TORCH_JIT_PATH), "wb") as file:
                 dill.dump(self, file)
         else:
+            shutil.rmtree(location)
```

Comment:
Will this call ever fail and cause another exception? I'm just wondering if the directory fails to create for some reason (permissions maybe), and then we try to delete it, will there be some issue?

Reply:
Good point! I think that if the directory creation fails, it will raise an exception on line 54. So at this point we should be safe to delete the dir here?

Reply:
I *think* that's right. But yah a test to double check would clear any doubts :)